sources/packages/forms/src/form-definitions/studentadditionaltransportationappeal.json

```diff
@@ -0,0 +1,1124 @@
+                  "tableView": false,
+                  "inputType": "text",
+                  "inputMask": "",
+                  "label": "Please estimate your weekly transportation costs.",
+                  "key": "additionalTransportCost",
+                  "placeholder": "",
+                  "prefix": "$",
+                  "suffix": "",
+                  "defaultValue": "",
+                  "protected": false,
+                  "persistent": true,
+                  "hidden": false,
+                  "clearOnHide": true,
+                  "delimiter": true,
+                  "decimalLimit": 2,
+                  "requireDecimal": true,
+                  "validate": {
+                    "required": true,
+                    "multiple": "",
+                    "custom": "valid = (input >= 0 && input <= 9999) ? true : 'The number you have entered is outside the expected range.';"
+                  },
+                  "conditional": {
+                    "show": "",
+                    "when": null,
+                    "eq": ""
+                  },
+                  "type": "currency",
+                  "labelPosition": "top",
+                  "tags": [],
+                  "properties": {},
+                  "lockKey": true,
+                  "customConditional": ""
+                },
+                {
+                  "autofocus": false,
+                  "input": true,
+                  "tableView": false,
+                  "inputType": "number",
+                  "label": "Please estimate your weekly travel distance in kilometers:",
+                  "key": "additionalTransportKm",
+                  "placeholder": "",
+                  "prefix": "",
+                  "suffix": "km",
+                  "defaultValue": "",
+                  "protected": false,
+                  "persistent": true,
+                  "hidden": false,
+                  "clearOnHide": true,
+                  "validate": {
+                    "required": true,
+                    "min": 0,
+                    "max": 999,
+                    "step": "any",
+                    "integer": "",
+                    "multiple": "",
+                    "custom": ""
+                  },
+                  "type": "number",
+                  "labelPosition": "top",
+                  "tags": [],
+                  "conditional": {
+                    "show": "",
+                    "when": null,
+                    "eq": ""
+                  },
+                  "properties": {},
+                  "decimalLimit": 0,
+                  "lockKey": true,
+                  "customConditional": ""
+                },
+                {
+                  "autofocus": false,
+                  "input": true,
+                  "tableView": false,
+                  "inputType": "number",
+                  "label": "How many weeks during your study period will you require additional transportation costs?",
```

Comment:
Not a part of the PR. Surprisingly additional transport weeks is not a mandatory field in student application and hence here. 

![image](https://github.com/bcgov/SIMS/assets/54600590/d43c13dd-22a1-40c8-929e-8c31665b9d6e)

Reply:
We can pass on the message to business and have them a ticket created for it.

Reply:
we have a fallback for that and I didn't see any calculation using that variable.

Reply:
The calculation is supposed to be using it, I believe that the fallback is doing the trick.
As @dheepak-aot mentioned, the business should confirm if it should be mandatory or not.

Reply:
We got the business confirmation from @Joshua-Lakusta that the field should be required.

Reply:
ok.Changed.
![image](https://github.com/bcgov/SIMS/assets/78114138/58fb313a-4cc1-4e77-8804-0755cc47e5d6)

Reply:
Thanks for doing the change.